IoT-Sample/Swift/IoTSampleSwift/ConnectionViewController.swift

```diff
@@ -19,207 +19,273 @@ import AWSMobileClient
 
 class ConnectionViewController: UIViewController, UITextViewDelegate {
 
-    @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
-    @IBOutlet weak var logTextView: UITextView!
-
-    @objc var connected = false;
-    @objc var publishViewController : UIViewController!;
-    @objc var subscribeViewController : UIViewController!;
-    @objc var configurationViewController : UIViewController!;
-
-    @objc var iotDataManager: AWSIoTDataManager!;
-    @objc var iotManager: AWSIoTManager!;
+    let IOT_CERT = "IoT Cert"
+    let IOT_WEBSOCKET = "IoT Websocket"
+
+    var connectIoTDataWebSocket: UIButton!
+    var activityIndicatorView: UIActivityIndicatorView!
+    var logTextView: UITextView!
+    var connectButton: UIButton!
+    
+    @objc var connected = false
+    @objc var publishViewController : UIViewController!
+    @objc var subscribeViewController : UIViewController!
+    @objc var configurationViewController : UIViewController!
+
+    @objc var iotDataManager: AWSIoTDataManager!
+    @objc var iotManager: AWSIoTManager!
     @objc var iot: AWSIoT!
 
-    @IBAction func connectButtonPressed(_ sender: UIButton) {
+    func mqttEventCallbackWebsocket( _ status: AWSIoTMQTTStatus ) {
+        if case .connected = status {
+            DispatchQueue.main.async {                
+                let tabBarViewController = self.tabBarController as! IoTSampleTabBarController
+                tabBarViewController.mqttStatus = "Connected"
+                self.activityIndicatorView.stopAnimating()
+                self.connected = true
+                self.connectIoTDataWebSocket.setTitle("Disconnect \(self.IOT_WEBSOCKET)", for:UIControl.State())
+                self.logTextView.text = "Connected via websocket"
+                self.connectIoTDataWebSocket.isEnabled = true
+                tabBarViewController.viewControllers = [ self, self.publishViewController, self.subscribeViewController ]
+            }
+        } else {
+            mqttEventCallback(status)
+        }
+    }
 
-        let tabBarViewController = tabBarController as! IoTSampleTabBarController
+    func mqttEventCallback( _ status: AWSIoTMQTTStatus ) {
+        DispatchQueue.main.async {
+            let tabBarViewController = self.tabBarController as! IoTSampleTabBarController
+            print("connection status = \(status.rawValue)")
+            switch(status)
+            {
```

Comment:
prefer opening braces inline with the statement. You can lose the parentheses while you're at it :)

Reply:
Done.